packages/components/src/Tooltip/Tooltip.tsx

```diff
@@ -65,11 +64,11 @@ export function Tooltip({ message, children }: TooltipProps) {
               exit="startOrStop"
               transition={{
                 type: "spring",
```

Comment:
just need to remove this and it will stop bouncing

Reply:
I modified the animation so it still _moves_, it's just not so "bouncy"

Reply:
oh, if you remove that, it will fallback to tween, which is the `ease` in css

Reply:
Do we need to remove it? It's working as I'd expect right now with the new damping and stiffness values

https://github.com/GetJobber/atlantis/assets/39704901/203a9129-bfad-4a75-b29c-4280ae6b1ddf

Reply:
OK so what is the recommended change? Do I need to delete the entire Framer Motion div, or just change some of these declarations?

Reply:
just need to remove the type

Reply:
Perfect, will do!